types/block.go

```diff
@@ -1251,14 +1247,24 @@ func (data *Data) ToProto() tmproto.Data {
 		}
 		tp.IntermediateStateRoots.RawRootsList = roots
 	}
-	// TODO(evan): copy missing pieces from previous work
+
 	pevd, err := data.Evidence.ToProto()
 	if err != nil {
 		// TODO(evan): fix
 		panic(err)
 	}
 	tp.Evidence = *pevd
 
+	// need to add messages
```

Comment:
This comment makes it sound like a TODO?

Reply:
good point, we should remove this https://github.com/celestiaorg/celestia-core/pull/580/commits/7f053bb043de85efeb8f58a6a431d47d5eff7247